x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/new_search_index_page.tsx

```diff
@@ -55,32 +55,26 @@ function getTitle(method: string, serviceType: string): string {
   }
 }
 
-function getDescription(method: string, serviceType: string): string {
+function getDescription(method: string): string {
   switch (method) {
     case INGESTION_METHOD_IDS.API:
       return i18n.translate('xpack.enterpriseSearch.content.new_index.apiDescription', {
-        defaultMessage: 'A search index stores your data.',
+        defaultMessage:
+          'Use the API to programatically add documents to an Elasticsearch index. Start by creating your index.',
       });
     case INGESTION_METHOD_IDS.CONNECTOR: {
-      const connector =
-        Boolean(serviceType) && CONNECTORS.find((item) => item.serviceType === serviceType);
-      return connector
-        ? i18n.translate(
-            'xpack.enterpriseSearch.content.new_index.connectorDescriptionWithServiceType',
-            {
-              defaultMessage: 'A search index stores the data for your {name} connector.',
-              values: {
-                name: connector.name,
-              },
-            }
-          )
-        : i18n.translate('xpack.enterpriseSearch.content.new_index.connectorDescription', {
-            defaultMessage: 'A search index stores the data for your connector.',
-          });
+      return i18n.translate(
+        'xpack.enterpriseSearch.content.new_index.connectorDescriptionWithServiceType',
+        {
+          defaultMessage:
+            'Use connectors to sync data from an original data source to an Elasticsearch index. Start by creating your index.',
+        }
```

Comment:
I propose to use a text more aligned with Crawler

> Use a connector to sync, extract, transform and index data from your data source. Connectors are a type of [Elastic integration](https://www.elastic.co/integrations/data-integrations) that sync data sources directly to Elasticsearch indices.

@leemthompo WDYT?

Reply:
Looks good, could make it slightly briefer:

_Use a connector to sync, extract, transform and index data from your data source. Connectors are an [Elastic integration]() that write directly to Elasticsearch indices._

Reply:
should the link point to https://www.elastic.co/guide/en/enterprise-search/master/connectors.html in that case ? @danajuratoni

Reply:
Good question, do we want to switch context by linking to the[ integrations docs](https://docs.elastic.co/integrations) here or should we avoid having a link here? I'd lean towards not having a link.

Reply:
We are not listing connectors on [integrations docs](https://docs.elastic.co/integrations) so the website or the in product integrations page would be the only viable options.

Saying "_Connectors are Elastic integrations that write directly to Elasticsearch indices._" without providing any more context on integrations feels a bit cryptic but I agree switching context might be too much info. Let's follow @leemthompo suggestion and keep it simple, without links.